packages/components/src/components/combobox/option-renderer.ts

```diff
@@ -26,6 +26,28 @@ export const highlightOptionRenderer: OptionRenderer = (option: SynOption, query
     return option;
   }
 
+  const combobox = option.closest('syn-combobox');
+  // check if there is already a style for highlighting appended if not append one
+  // This is unfortunately needed as it is not possible to do something like this in css:
+  // .listbox__options ::slotted(syn-option mark)
+  if (combobox && !combobox?.querySelector('#syn-highlight-style')) {
```

Comment:
I understand where this is coming from. However, could we add this to the `<syn-option>` shadow root instead? Did not test it, but we could also add a custom marker, e.g. `<mark class="highlighted-element">` and use this in the `syn-option` as `.label::slotted(.highlighted-element`)?

As it is, the style is not overridable from the outside, not even with important. Don´t think this is a big problem, fwiw, so we may also go ahead with this approach.

Reply:
I like the idea 👍
I was only thinking about doing something like this in the option style `.option__label::slotted(mark)`. But I didn't like that we automatically style the `<mark/>` element always.
Was not thinking about i could just add a class for this instead.